js/main.js

```diff
@@ -0,0 +1,83 @@
+document.addEventListener( "DOMContentLoaded", function() {
+
+    let boxes = document.querySelectorAll("#big-box div");
```

Comment:
jeśli nie zamierzasz dalej w kodzie nadpisywać tej zmiennej to zamiast let używaj const. Jeden ze wzorców mówi, że jeśli przechowujesz referencję do obiektu z drzewa DOM to zmienna powinna się zaczynać od $

Reply:
jeśli chodzi o #big-box div to lepiej danemu div nadać klasę która będzie odpowiednia dla jego funkcji